pkgs/by-name/bi/bitmap/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  pkg-config,
+  util-macros,
+  autoreconfHook,
+  wrapWithXFileSearchPathHook,
+  libx11,
+  libxaw,
+  xbitmaps,
+  libxmu,
+  xorgproto,
+  libxt,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bitmap";
+  version = "1.1.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "xorg";
+    repo = "app/bitmap";
```

We have group for cases like this:

```suggestion
    group = "xorg";
    owner = "app";
    repo = "bitmap";
```

---

done